app/src/main/java/com/zulip/android/ZulipApp.java

```diff
@@ -70,6 +72,13 @@ public void onCreate() {
         lastEventId = settings.getInt("lastEventId", -1);
         pointer = settings.getInt("pointer", -1);
 
+
+        if (Build.HARDWARE.contains("goldfish") || BuildConfig.DEBUG) { // dont load crashlytics for debug builds
```

Comment:
Inline comments are generally discouraged — can this be moved to the line above?